errors/Readme.md

```diff
@@ -20,40 +20,6 @@ Definition of errors in Prisma Framework. (In this document we make the distinct
 - [Known Errors](#known-errors)
   - [Known Errors Template](#known-errors-template)
   - [Prisma SDK](#prisma-sdk)
-    - [Common](#common)
-      - [P1000: Incorrect database credentials](#p1000-incorrect-database-credentials)
-      - [P1001: Database not reachable](#p1001-database-not-reachable)
-      - [P1002: Database timeout](#p1002-database-timeout)
-      - [P1003: Database does not exist](#p1003-database-does-not-exist)
-      - [P1004: Incompatible binary](#p1004-incompatible-binary)
-      - [P1005: Unable to start the query engine](#p1005-unable-to-start-the-query-engine)
-      - [P1006: Binary not found](#p1006-binary-not-found)
-      - [P1007: Missing write access to download binary](#p1007-missing-write-access-to-download-binary)
-      - [P1008: Database operation timeout](#p1008-database-operation-timeout)
-      - [P1009: Database already exists](#p1009-database-already-exists)
-      - [P1010: Database access denied](#p1010-database-access-denied)
-      - [P1011: TLS connection error](#p1011-tls-connection-error)
-    - [Query Engine](#query-engine)
-      - [P2000: Input value too long](#p2000-input-value-too-long)
-      - [P2001: Record not found](#p2001-record-not-found)
-      - [P2002: Unique key violation](#p2002-unique-key-violation)
-      - [P2003: Foreign key violation](#p2003-foreign-key-violation)
-      - [P2004: Constraint violation](#p2004-constraint-violation)
-      - [P2005: Stored value is invalid](#p2005-stored-value-is-invalid)
-      - [P2006: `*`Type mismatch: invalid (ID/Date/Json/Enum)](#p2006-type-mismatch-invalid-iddatejsonenum)
-      - [P2007: `*`Type mismatch: invalid custom type](#p2007-type-mismatch-invalid-custom-type)
-      - [P2008: Query parsing failed](#p2008-query-parsing-failed)
-      - [P2009: Query validation failed](#p2009-query-validation-failed)
-    - [Migration Engine](#migration-engine)
-      - [P3000: Database creation failed](#p3000-database-creation-failed)
-      - [P3001: Destructive migration detected](#p3001-destructive-migration-detected)
-      - [P3002: Migration rollback](#p3002-migration-rollback)
-    - [Introspection Engine](#introspection-engine)
-      - [P4000: Introspection failed](#p4000-introspection-failed)
-    - [Schema Parser](#schema-parser)
```

Comment:
Schema parser errors don't really exist or are now just part of something else?

Reply:
We did not implement this part of the spec. The three listed errors were too coarse grained and incomplete. I think we should make a separate effort to define those properly when we actually need them. Therefore i removed them altogether.